dotnet/src/SemanticKernel.Abstractions/Orchestration/IKernelExecutionContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel.Orchestration;
+
+/// <summary>
+/// Kernel execution context
+/// </summary>
+public interface IKernelExecutionContext
+{
+    /// <summary>
+    /// App logger
+    /// </summary>
+    ILoggerFactory LoggerFactory { get; }
+
+    /// <summary>
+    /// Read only skills collection
+    /// </summary>
+    IReadOnlyFunctionCollection Functions { get; }
```

Comment:
SKContext already has Functions property. I would propose to use it instead of adding another one.

Reply:
Removed from the interface